packages/dapp-simple-exchange/contract/deploy-autoswap.js

```diff
@@ -132,15 +132,13 @@ export default async function deployContract(
   async function addExchangeOffer(offerAmount, payments, wantAmount) {
     const offerRules = makeRulesOfferXForY(offerAmount, wantAmount);
     const { invite: anInvite } = await publicAPI~.makeInvite();
-    const { seat } = await homeP~.zoe~.redeem(anInvite, offerRules, payments);
-    return seat~.addOrder();
+    return homeP~.zoe~.redeem(anInvite, offerRules, payments)~.offerResult;
```

Comment:
I think this is outside the scope of this PR

Reply:
Look again. The old code redeems and calls `addOrder`. The new code redeems and uses `offerResult`.